### ISSUE TITLE
German language support

### ISSUE THREAD

**Issue by XP312** [akiraux/Akira#427]
I take time and made **de.po** file to support German and is it right that the language 
support button links to the Issues Page? 
Some by 5 things I'm not sure if it's right translated, do i marked it so.

de.po:

```
#
# XP312 < >, 2020.
#
msgid ""
msgstr ""
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"Project-Id-Version: unnamed project\n"
"Last-Translator: XP312 < >\n"
"Language-Team: German < >\n"
"Language: de\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2020-08-26 19:39+0200\n"
"X-Generator: Gtranslator 3.36.0\n"

#: Dialogs/ExportDialog.vala:151
msgid "Export to:"
msgstr "Exportieren zu:"

#: Dialogs/ExportDialog.vala:157
msgid "Select Folder"
msgstr "Ordner auswählen"

#: Dialogs/ExportDialog.vala:166
msgid "Format:"
msgstr "Format:"

#: Dialogs/ExportDialog.vala:180
msgid "Quality:"
msgstr "Qualität:"

#: Dialogs/ExportDialog.vala:192
msgid "Compression:"
msgstr "Kompremieren:"

#: Dialogs/ExportDialog.vala:206
msgid "Transparency:"
msgstr "Transperenz:"

#: Dialogs/ExportDialog.vala:219
msgid "Scale:"
msgstr "Größe:"

#: Dialogs/ExportDialog.vala:243 FileFormat/FileManager.vala:49
#: FileFormat/FileManager.vala:105
msgid "Cancel"
msgstr "Abrechen"

#: Dialogs/ExportDialog.vala:250 Dialogs/ShortcutsDialog.vala:63
#: Layouts/HeaderBar.vala:112
msgid "Export"
msgstr "Exportieren"

#: Dialogs/ReleaseDialog.vala:47 Dialogs/SettingsDialog.vala:217
msgid ""
"WARNING!\n"
"Akira is still under development and not ready for production. Missing "
"features, random bugs, and black holes opening in your kitchen are to be "
"expected."
msgstr ""
"ACHTUNG:\n"
"Akira ist noch immer in Arbeit und ist noch nicht fertig für produktion. "
"Fehlende Funktionen, Bugs und Schwarzelöcher die sich in der Küche öffnen "
"sind zu erwarten."

#: Dialogs/ReleaseDialog.vala:100 Dialogs/SettingsDialog.vala:249
msgid "Make a Donation"
msgstr "Spendet!"

#: Dialogs/ReleaseDialog.vala:109 Dialogs/SettingsDialog.vala:258
msgid "Suggest Translations"
msgstr "Schlagt Übersetzungen vor"

#: Dialogs/ReleaseDialog.vala:118 Dialogs/SettingsDialog.vala:267
msgid "Report a Problem"
msgstr "Meldet Fehler"

#: Dialogs/SettingsDialog.vala:40
msgid "Preferences"
msgstr "Einstellungen"

#: Dialogs/SettingsDialog.vala:50 Dialogs/SettingsDialog.vala:73
#: Dialogs/ShortcutsDialog.vala:45
msgid "General"
msgstr "Generell"

#: Dialogs/SettingsDialog.vala:51 Dialogs/SettingsDialog.vala:86
msgid "Interface"
msgstr "Oberfläche"

#: Dialogs/SettingsDialog.vala:52 Layouts/HeaderBar.vala:280
msgid "Shapes"
msgstr "Formen"

#: Dialogs/SettingsDialog.vala:53
msgid "About"
msgstr "Über"

#: Dialogs/SettingsDialog.vala:74
msgid "Auto Reopen Latest File:"
msgstr "Automatisch die letzte Datei öffnen:"

#: Dialogs/SettingsDialog.vala:88
msgid "Use Dark Theme:"
msgstr "Schwarzes Thema nutzen:"

#: Dialogs/SettingsDialog.vala:92
msgid "Invert Panels Order:"
msgstr "Seiten Felder Ordnung umdrehen:"

#: Dialogs/SettingsDialog.vala:96
msgid "Restart application to apply this change."
msgstr "Neustarten der Anwendungen um Änderungen anzuwenden."

#: Dialogs/SettingsDialog.vala:100
msgid "ToolBar Style"
msgstr "Werkzeugleisten Ausehen"

#: Dialogs/SettingsDialog.vala:102
msgid "Show Button Labels:"
msgstr "Knopf Benenung anzeigen:"

#: Dialogs/SettingsDialog.vala:106
msgid "Use Symbolic Icons:"
msgstr "Symbolische Icons nutzen:"

#: Dialogs/SettingsDialog.vala:129
msgid "Default Colors"
msgstr "Vorgabe Farben"

#: Dialogs/SettingsDialog.vala:131
msgid "Define the default style used when creating a new shape."
msgstr "Defeniere das standard Ausehen einer neuen Form"

#: Dialogs/SettingsDialog.vala:136
msgid "Fill Color:"
msgstr "Füllfarbe:"

#: Dialogs/SettingsDialog.vala:157
msgid "Enable Border Style:"
msgstr "Akteviere Umransungs ausehen:"

#: Dialogs/SettingsDialog.vala:160
msgid "Border Color:"
msgstr "Umrandungslinien Farbe:"

#: Dialogs/SettingsDialog.vala:181
msgid "Border Width:"
msgstr "Umrandungslinien Dicke:"

#: Dialogs/SettingsDialog.vala:209
msgid "The Linux Design Tool"
msgstr "Das Linux Design Werkzeug"

#: Dialogs/SettingsDialog.vala:226
msgid "Thanks to our awesome supporters!"
msgstr "Danke an unseren tollen Unterstützern"

#: Dialogs/SettingsDialog.vala:231
msgid "View the list of supporters"
msgstr "Liste aller Unterstützern"

#: Dialogs/ShortcutsDialog.vala:46
msgid "New window:"
msgstr "Neues Fenster:"

#: Dialogs/ShortcutsDialog.vala:48
msgid "Preferences:"
msgstr "Einstellungen:"

#: Dialogs/ShortcutsDialog.vala:50
msgid "Quit:"
msgstr "Schließen:"

#: Dialogs/ShortcutsDialog.vala:52
msgid "Presentation mode:"
msgstr "Präsentations Modus:"

#: Dialogs/ShortcutsDialog.vala:55
msgid "File"
msgstr "Datei"

#: Dialogs/ShortcutsDialog.vala:56
msgid "Open:"
msgstr "Öffnen:"

#: Dialogs/ShortcutsDialog.vala:58
msgid "Save:"
msgstr "Speichern:"

#: Dialogs/ShortcutsDialog.vala:60
msgid "Save as:"
msgstr "Speichern als:"

#: Dialogs/ShortcutsDialog.vala:64
msgid "Export artboards:"
msgstr "Exportiere das Zeichenbrett:"

#: Dialogs/ShortcutsDialog.vala:66
msgid "Export selection:"
msgstr "Exportiere das Selectierte:"

#: Dialogs/ShortcutsDialog.vala:68
msgid "Highlight area to export:"
msgstr "Wähle den Bereich fürs exportieren:"

#: Dialogs/ShortcutsDialog.vala:77
msgid "Canvas"
msgstr "Leinwand"

#: Dialogs/ShortcutsDialog.vala:78
msgid "Zoom in:"
msgstr "Vergrößern"

#: Dialogs/ShortcutsDialog.vala:80
msgid "Zoom out:"
msgstr "Verkleinern"

#: Dialogs/ShortcutsDialog.vala:82
msgid "Zoom reset:"
msgstr "Ansicht zurück setzen"

#: Dialogs/ShortcutsDialog.vala:85
msgid "Item creation"
msgstr "Objekt erstellen"

#: Dialogs/ShortcutsDialog.vala:86
msgid "Artboard:"
msgstr "Zeichenbrett:"

#: Dialogs/ShortcutsDialog.vala:88
msgid "Rectangle:"
msgstr "Rechteck:"

#: Dialogs/ShortcutsDialog.vala:90
msgid "Ellipse:"
msgstr "Oval:"

#: Dialogs/ShortcutsDialog.vala:92
msgid "Text:"
msgstr "Text:"

#: Dialogs/ShortcutsDialog.vala:94
msgid "Image:"
msgstr "Foto:"

#: Dialogs/ShortcutsDialog.vala:97 Layouts/Partials/TransformPanel.vala:174
msgid "Transform"
msgstr "Transformieren"

#: Dialogs/ShortcutsDialog.vala:98
msgid "Raise selection:"
msgstr "Hoch:"

#: Dialogs/ShortcutsDialog.vala:100
msgid "Lower selection:"
msgstr "Runter:"

#: Dialogs/ShortcutsDialog.vala:102
msgid "Raise selection to top:"
msgstr "Nach Oben:"

#: Dialogs/ShortcutsDialog.vala:104
msgid "Lower selection to bottom:"
msgstr "Nach Hinten:"

#: Dialogs/ShortcutsDialog.vala:106
msgid "Flip horizontally:"
msgstr "Horizontal drehen:"

#: Dialogs/ShortcutsDialog.vala:108
msgid "Flip vertically:"
msgstr "Vertikal drehen:"

#: FileFormat/FileManager.vala:46
msgid "Save Akira file"
msgstr "Akiradatei speichern:"

#: FileFormat/FileManager.vala:48 Layouts/HeaderBar.vala:190
msgid "Save"
msgstr "Speichern:"

#: FileFormat/FileManager.vala:71
msgid "Akira files"
msgstr "Akiradatei"

#: FileFormat/FileManager.vala:76
msgid "All files"
msgstr "Alle Datein"

#: FileFormat/FileManager.vala:103
msgid "Open Akira file"
msgstr "Akiradatei öffnen"

#: FileFormat/FileManager.vala:105 Layouts/HeaderBar.vala:170
msgid "Open"
msgstr "Öffnen"

#: Layouts/HeaderBar.vala:69 Lib/Managers/ExportManager.vala:296
msgid "Untitled"
msgstr "Ungespeichert"

#: Layouts/HeaderBar.vala:71
msgid "Menu"
msgstr "Menü"

#: Layouts/HeaderBar.vala:75
msgid "Insert"
msgstr "Einfügen"

#: Layouts/HeaderBar.vala:82
msgid "Group"
msgstr "Gruppieren"

#: Layouts/HeaderBar.vala:84
msgid "Ungroup"
msgstr "Gruppierung auflösen"

#: Layouts/HeaderBar.vala:87
msgid "Up"
msgstr "Hoch"

#: Layouts/HeaderBar.vala:92
msgid "Down"
msgstr "Runter"

#: Layouts/HeaderBar.vala:97
msgid "Top"
msgstr "Nach Oben"

#: Layouts/HeaderBar.vala:102
msgid "Bottom"
msgstr "Nach Hinten"

#: Layouts/HeaderBar.vala:107
msgid "Settings"
msgstr "Einstellungen"

# ?
#: Layouts/HeaderBar.vala:118
#, fuzzy
msgid "Difference"
msgstr "Gegensatz"

#: Layouts/HeaderBar.vala:120
#, fuzzy
msgid "Exclusion"
msgstr "Ausgrenzen"

#: Layouts/HeaderBar.vala:122
#, fuzzy
msgid "Intersect"
msgstr "Übergehend"

#: Layouts/HeaderBar.vala:124
msgid "Union"
msgstr "Verbinden"

#: Layouts/HeaderBar.vala:161
msgid "New Window"
msgstr "Neues Fenster"

#: Layouts/HeaderBar.vala:182
msgid "Open Recent"
msgstr "Vorheriges öffnen "

#: Layouts/HeaderBar.vala:195
msgid "Save As"
msgstr "Speichern als"

#: Layouts/HeaderBar.vala:204
msgid "Quit"
msgstr "Schließen"

#: Layouts/HeaderBar.vala:237
msgid "Artboard"
msgstr "Zeichenbrett"

#: Layouts/HeaderBar.vala:254
msgid "Add Items"
msgstr "Objekt hinzufügen"

#: Layouts/HeaderBar.vala:262
msgid "Rectangle"
msgstr "Rechteck"

#: Layouts/HeaderBar.vala:268
msgid "Ellipse"
msgstr "Oval"

#: Layouts/HeaderBar.vala:286
msgid "Vector"
msgstr "Vektor"

#: Layouts/HeaderBar.vala:288
msgid "Pencil"
msgstr "Stift"

#: Layouts/HeaderBar.vala:291
msgid "Text"
msgstr "Text"

#: Layouts/HeaderBar.vala:297
msgid "Image"
msgstr "Foto"

#: Layouts/HeaderBar.vala:330
msgid "Export Current Selection"
msgstr "Exportiert Selektiertes"

#: Layouts/HeaderBar.vala:336
msgid "Export Artboards"
msgstr "Exportiert aktuelles Zeichenbrett"

#: Layouts/HeaderBar.vala:345
msgid "Highlight Area to Export"
msgstr "Exportiert den ausgewählte Bereich"

#: Layouts/HeaderBar.vala:391
msgid "Main Menu"
msgstr "Hauptmenü"

#: Layouts/HeaderBar.vala:468 Services/ActionManager.vala:218
#: Services/ActionManager.vala:237 Services/ActionManager.vala:256
#, c-format
msgid "Unable to open file at '%s'"
msgstr "Speichern von \"%s\""

#: Layouts/MainCanvas.vala:52
msgid "Button was pressed!"
msgstr "Knopf war gedrückt!"

#: Layouts/MainCanvas.vala:160
msgid "Export completed!"
msgstr "Exportartion fertigstellt."

#: Layouts/Partials/AlignItemsPanel.vala:48
#, fuzzy
msgid "Distribute Horizontally"
msgstr "Verteile Horizontall"

#: Layouts/Partials/AlignItemsPanel.vala:50
#, fuzzy
msgid "Distribute Vertically"
msgstr "Verteile Vertikal "

#: Layouts/Partials/AlignItemsPanel.vala:52
msgid "Align Left"
msgstr "Links ausrichten"

#: Layouts/Partials/AlignItemsPanel.vala:53
msgid "Align Center"
msgstr "Zentral ausrichten"

#: Layouts/Partials/AlignItemsPanel.vala:54
msgid "Align Right"
msgstr "Rechts Ausrichten"

#: Layouts/Partials/AlignItemsPanel.vala:56
msgid "Align Top"
msgstr "Nach Oben ausrichten"

#: Layouts/Partials/AlignItemsPanel.vala:57
msgid "Align Middle"
msgstr "Mittig ausrichten"

#: Layouts/Partials/AlignItemsPanel.vala:58
msgid "Align Bottom"
msgstr "Nach unten ausrichten"

#: Layouts/Partials/Artboard.vala:144 Layouts/Partials/Artboard.vala:487
#: Layouts/Partials/Layer.vala:139 Layouts/Partials/Layer.vala:687
msgid "Lock Layer"
msgstr "Ebene sperren"

#: Layouts/Partials/Artboard.vala:155 Layouts/Partials/Artboard.vala:529
#: Layouts/Partials/Layer.vala:156 Layouts/Partials/Layer.vala:729
msgid "Hide Layer"
msgstr "Ebene Verstecken"

#: Layouts/Partials/Artboard.vala:487 Layouts/Partials/Layer.vala:687
msgid "Unlock Layer"
msgstr "Ebene entsperren"

#: Layouts/Partials/Artboard.vala:529 Layouts/Partials/Layer.vala:729
msgid "Show Layer"
msgstr "Ebene Zeigen"

#: Layouts/Partials/BorderRadiusPanel.vala:86
msgid "Style"
msgstr "Ausehen"

#: Layouts/Partials/BorderRadiusPanel.vala:103
msgid "Border Radius"
msgstr "Ecken Abrunden"

#: Layouts/Partials/BorderRadiusPanel.vala:198
msgid "Autoscale Corners"
msgstr "Ecken Automatisch skallieren"

#: Layouts/Partials/BorderRadiusPanel.vala:207
msgid "Uniform Corners"
msgstr "Ecken angleichen"

#: Layouts/Partials/BordersPanel.vala:53
msgid "Borders"
msgstr "Grenzen"

#: Layouts/Partials/FillsPanel.vala:54
msgid "Fills"
msgstr "Datein"

#: Layouts/Partials/TransformPanel.vala:96
msgid "X"
msgstr "X"

#: Layouts/Partials/TransformPanel.vala:96
msgid "Horizontal position"
msgstr "Horizontale Position"

#: Layouts/Partials/TransformPanel.vala:98
msgid "Y"
msgstr "Y"

#: Layouts/Partials/TransformPanel.vala:98
msgid "Vertical position"
msgstr "Vertikale Position"

#: Layouts/Partials/TransformPanel.vala:100
msgid "W"
msgstr "W"

#: Layouts/Partials/TransformPanel.vala:100
msgid "Width"
msgstr "Breite"

#: Layouts/Partials/TransformPanel.vala:102
msgid "H"
msgstr "H"

#: Layouts/Partials/TransformPanel.vala:102
msgid "Height"
msgstr "Höhe"

#: Layouts/Partials/TransformPanel.vala:107
msgid "Lock Ratio"
msgstr "Rotation sperren"

#: Layouts/Partials/TransformPanel.vala:114
msgid "R"
msgstr "R"

#: Layouts/Partials/TransformPanel.vala:114
#, fuzzy
msgid "Rotation degrees"
msgstr "Rotation"

#: Layouts/Partials/TransformPanel.vala:126
msgid "Flip Horizontally"
msgstr "Horizontahl Drehen"

#: Layouts/Partials/TransformPanel.vala:137
msgid "Flip Vertically"
msgstr "Vertikal Drehen"

#: Layouts/Partials/TransformPanel.vala:165
msgid "Position"
msgstr "Position"

#: Layouts/Partials/TransformPanel.vala:169
msgid "Size"
msgstr "Größe"

#: Layouts/Partials/TransformPanel.vala:178
msgid "Opacity"
msgstr "Durchsichtigkeit"

#: Layouts/RightSideBar.vala:111
msgid "Search Layer"
msgstr "Suche Ebene"

#: Lib/Managers/ExportManager.vala:192 Lib/Managers/ExportManager.vala:219
msgid "Generating preview, please wait…"
msgstr "Generiere Vorschau, bitte warten"

#: Lib/Managers/ExportManager.vala:313
#, c-format
msgid "Untitled %i"
msgstr "Unbenannt %i"

#: Lib/Managers/ExportManager.vala:484
msgid "Exporting images…"
msgstr "Exportiere Fotos..."

#: Partials/ExportWidget.vala:117
#, c-format
msgid "%i × %i px · %s"
msgstr "%i × %i px · %s"

#: Partials/ExportWidget.vala:124
msgid "Fetching image size…"
msgstr "Lese Foto Größe aus..."

#: Partials/ZoomButton.vala:49
msgid "Zoom Out"
msgstr "Verkleinern"

#: Partials/ZoomButton.vala:55
msgid "Reset Zoom"
msgstr "Ansicht zurücksetzen"

#: Partials/ZoomButton.vala:62
msgid "Zoom In"
msgstr "Vergrößern"

#: Partials/ZoomButton.vala:68
msgid "Zoom"
msgstr "Vergrößerung"

#: Services/ActionManager.vala:211
msgid "No recently opened file available!"
msgstr "Keine zuletzt geöffnete Datei vorhanden!"

#: Services/ActionManager.vala:230
msgid "No second most recently opened file available!"
msgstr "Keine zweit zuletzt geöffnete Datei vorhanden!"

#: Services/ActionManager.vala:249
msgid "No third most recently opened file available!"
msgstr "Keine dritt zuletzt geöffnete Datei vorhanden!"

#: Services/ActionManager.vala:287
msgid "Nothing selected to export!"
msgstr "Nichts zu Exportieren!"

#: Services/ActionManager.vala:296
msgid "Export of Artboards currently unavailable…sorry 😑️"
msgstr "Exportieren von Zeichenbrettern ist leider momentan nicht möglich… 😑️"

#: Services/ActionManager.vala:363
msgid "Choose image file"
msgstr "Wähle eine Bilddatei"

#: Services/ActionManager.vala:363
msgid "Select"
msgstr "Selectiere"

#: Services/ActionManager.vala:363
msgid "Close"
msgstr "Schließen"

#: Services/ActionManager.vala:432
#, c-format
msgid "Error! .%s files are not supported!"
msgstr "Error: Diese Datei .%s ist nicht Unterstützt."

#: Window.vala:119
msgid "Are you sure you want to quit?"
msgstr "Willst du wirklich schließen?"

#: Window.vala:120
msgid "All unsaved data will be lost and impossible to recover."
msgstr "Alles nicht gespeicherte wird unwiderruflich gelöscht sein."

#: Window.vala:122
msgid "Quit without saving!"
msgstr "Schließen ohne speichern!"

#: Window.vala:123
msgid "Save file"
msgstr "Datei speichern"
```

**Comment by albfan** [comment]
Can you add that files to repo and create a PR?

**Comment by tenzens** [comment]
I am german and am happy that you have put in the effort to make german translations, so thank you!
However I would suggest you to go over the words again. I can see some incorrect writings.
For example: its "Exportieren nach", "Komprimieren", "Transparenz", "Abbrechen", and some other words. Maybe try and check your translation with google translate or deepl.org. But still thank you!

**Comment by albfan** [comment]
@tenzens. We are not native german speakers, so anything will work. We relay on you to review it.

**Comment by XP312** [comment]
@tenzens I German too but that does not mean I'm perfect I will control it know once more. I probably skip this words accidentally. And rewrite the wrong parts.

**Comment by XP312** [comment]
PR stand for Pull Request?

**Comment by nisrulz** [comment]
@XP312 Yes.

**Comment by codedracula4ec** [comment]
@XP312 yes. I made a pt_BR.po. I'm looking how to PR

**Comment by Alecaddd** [comment]
Fixed by #437

**Comment by JVariance** [comment]
There are many misspellings already in the first few translations.
If I find time, I'll take a look into it.
